core/src/main/java/org/mobilitydata/gtfsvalidator/annotation/PrimaryKey.java

```diff
@@ -32,20 +32,20 @@
  * <p>Example.
  *
  * <pre>
- *   @GtfsTable("stops.txt")
+ *   {@literal @}GtfsTable("stops.txt")
  *   public interface GtfsStopSchema extends GtfsEntity {
- *       @FieldType(FieldTypeEnum.ID)
- *       @Required
- *       @PrimaryKey
+ *       {@literal @}FieldType(FieldTypeEnum.ID)
+ *       {@literal @}Required
+ *       {@literal @}PrimaryKey
  *       String stopId();
  *   }
  * </pre>
  *
  * <p>This also adds a method to the container class to find an entity by its primary key.
  *
  * <pre>
- *   @Generated
- *   public class GtfsStopTableContainer extends GtfsTableContainer<GtfsStop> {
+ *   {@literal @}Generated
```

Comment:
@aababilov Do any of these Javadoc changes cause issues for you at Google? 

As @lionel-nj states in https://github.com/MobilityData/gtfs-validator/pull/718#issue-567114622 these are required to get the Javadocs to generate and render correctly. @lionel-nj looked at the Google Java Style Guide and it doesn't seem to cover this type of formatting, and as https://reflectoring.io/howto-format-code-snippets-in-javadoc/ indicates there are different ways to solve this problem. It looks like the changes @lionel-nj made here resolve the issue and can be merged, providing they aren't problematic for you.